source/modules/management/licenses.rst

```diff
@@ -0,0 +1,113 @@
+Licenses
+========
+
+Licenses management in GLPI allows to manage software licenses:
+
+* Inventorying licenses, with links to softwares inventoried in GLPI;
+* Following license installations on all assets;
+* Linking licenses to financial management;
+* Anticipating and following licenses renewal.
+
+.. note:: 
+        
+   A license cannot exists in GLPI without a software associated to the license when creating it.
+
+   License management is not automated, a human following is needed for informations update.
```

Comment:
```suggestion
   License management is not automated, a human follow-up is needed for information update.
```
Or
License management is not automated, a manual follow-up is needed for information update.